src/Templates/src/templates/maui-blazor/.template.config/template.json

```diff
@@ -12,6 +12,13 @@
       "type": "solution"
     },
     "sourceName": "MauiApp1",
+    "sources": [
+      {
+        "source": "./",
+        "target": "./",
+        "copyOnly": [ "**/wwwroot/css/**", "**/*.svg", "**/*.ttf" ]
```

Comment:
@Eilon let me know if you see anything else in the maui-blazor template that needs this.

Related: https://github.com/dotnet/aspnetcore/pull/33808